### ISSUE TITLE
Develop functionality to generate a spatially defined dashboard 

### ISSUE THREAD

**Issue by amyburness** [kartoza/django-bims#3609]
Develop functionality to generate a spatially defined dashboard that provides summary data.

**Comment by helendallas** [comment]
Feedback from SANParks - this would be very nice to have but chemists #3608 is more NB and should be completed first.

Di and Judith to provide a word doc and template of what the checklist needs to look like and what it needs to include.  Need to wait until this is received.

**Comment by helendallas** [comment]
As mentioned in #3608 a spatially defined dashboard would be very useful for FBIS as well. I may be able to provide some extra funds for this depending on how long you think it would take to develop.

Essentially we would use what we could from the current multisite dashboard but include summarised data in Overview, to allow a user to see all biodiversity data for a selected spatial area on one detailed dashboard.  I suggest including all graphs except to Taxa one, as this would be quite cumbersome, although it can be retained if it is easier.

I suggest this be called on the Side Panel, and it is positioned below the individual biodiversity dashboards and before the SASS Dashboard.

**Summary Dashboard**

Here is a mock layout. I have tried to keep most things the same as for other detailed dashboards except for the Overview which provides a summary of all biodiversity groups.

![Screenshot 2024-06-13 at 10 51 13](https://github.com/kartoza/django-bims/assets/45286876/f0b11189-ed16-40b9-9ffd-ba6bef691d58)


Details of Overview section -  we could use module icons instead of text, and this should be downloadable

![Screenshot 2024-06-13 at 10 52 07](https://github.com/kartoza/django-bims/assets/45286876/0970ca61-74ca-4acd-818c-be816e909bca)

**Comment by helendallas** [comment]
This is a very nice to have

**Comment by helendallas** [comment]
@jeremyshelton - please go though this and check what needs to be retained for FBIS, and FBIS Africa

**Comment by helendallas** [comment]
Reallocating this to FBIS Africa project

@jeremyshelton and @helendallas need to finalise on the details to be included in this dashboard prior to implementation

**Comment by jeremyshelton** [comment]
@helendallas OK will do.

**Comment by jeremyshelton** [comment]
@helendallas are you able to provide a higher quality screenshot, or perhaps the ppt file, if that's what you used to create the image? Thanks

**Comment by jeremyshelton** [comment]
@helendallas for the overview section: I think what you've done looks good for FBIS SA.
For FBIS Africa, the headings would just be: Freshwater Fish,, Amphibia, Odonata, Mollusca, Crustacea....and then the only category would be "IUCN Red List Status"

**Comment by helendallas** [comment]
The headers would pull directly from the Biodiversity Modules. It needs to be generic to be used across all BIMS, including SANParks.

@dimasciput We would need to specify which data were visible, so for FBIS, Origin, endemism and Global Conservation Status  and National Conservation Status would be relevant, but for FBIS Africa only Global Conservation Status  would be needed.

Here are the two files

[Summary Dashboard.docx](https://github.com/user-attachments/files/20189806/Summary.Dashboard.docx)
[SUmmary Dashboard Overview Table.xlsx](https://github.com/user-attachments/files/20189805/SUmmary.Dashboard.Overview.Table.xlsx)

**Comment by helendallas** [comment]
@dimasciput please have a look at this ticket and see if you have enough to generate a summary dashboard draft. Thanks

**Comment by jeremyshelton** [comment]
Looking good!

A few points:
- The dashboard name 'Spatial Dashboard' is not ver self-explanatory? Maybe 'Summary Dashboard' is a better name for this?
- I would change "Conservation Status Per Module" to "IUCN Conservation Status" (It's obvious fro the graph that it is per module, and I'm not sure if users know what 'Module' means?) (Or simply "Conservation Status" to be consistent").
- Table on the left: Conservation Status National should be removed for FBIS Africa. I think that "IUCN Conservation Status" makes more sense than "Conservation Status Global" in this context? (Or simply "Conservation Status" to be consistent").
- The Red List Index chart at the bottom - is that a SANBI/South Africa index, or a global index? If SANBI, would it apply to FBIS Africa?

**Comment by dimasciput** [comment]
@nazley-liddle updated in staging.

**Comment by nazley-liddle** [comment]
> [@dimasciput](https://github.com/dimasciput) please make the following changes:
> 
> * Under Occurrence charts, rename the 'Conservation Status' pie chart to 'Conservation Status Global'. This applies to all these graphs and headers across all dashboards. Naming must be consistent.

Looks good

> * Reorder the graphs to match the other dashboards: it should be Occurrences, Origin, Endemism, Conservation Status Global

Happy with the order on staging. 

@dimasciput please deploy these fixes to production. The graph edits can be handled in the other SANParks ticket.

**Comment by helendallas** [comment]
@dimasciput The species csv is not that user friendly as it is . Is it possible to have the taxon hierarchy, and then  origin, endemism, conservation status global, conservation status national, invasion/National NEMBA Status, Sorted by 
Kingdom	Phylum	Class	Order	Family	SubFamily	Tribe	SubTribe	Genus	SubGenus	Species	SubSpecies	Variety	SpeciesGroup	Taxon

<img width="858" height="532" alt="Image" src="https://github.com/user-attachments/assets/0b086440-0fff-4ba1-a96e-3d05026ef7d8" />
